haystack/nodes/retriever/multimodal/retriever.py

```diff
@@ -221,3 +221,8 @@ def retrieve_batch(  # type: ignore
 
     def embed_documents(self, docs: List[Document]) -> np.ndarray:
         return self.document_embedder.embed(documents=docs)
+
+    def embed_queries(self, queries: List[str]) -> np.ndarray:
+        raise NotImplementedError(
+            "MultiModal retriever does not support query embedder, use MultiModalEmbedder to embed the queries."
+        )
```

Comment:
This is supported actually, just hack it around for now :grin: 

```suggestion
        query_documents = =[Document(content=query, content_type="text") for query in queries]
        return [result.embedding for result in self.query_embedder.embed(documents=query_documents)]
```

Reply:
Done!